modules/ROOT/pages/document-types.adoc

```diff
@@ -27,7 +27,6 @@ Valid values:
 Default:
```

Comment:
Line 1
Do not use parentheses in topic titles. Choose the best term and then add the modification/explanation/alternative in the text that immediately follows the title (as you have done already):
"Message Types"

Reply:
Line 6
However, that said, to what does "that document standard" refer? Which document standard?

Reply:
Also, GLOBALLY evaluate the text that immediately follows each of your topic titles. It should meet these requirements:

* Address the audience as "you."
* Tell "you" why they want or need the following information: to what purpose, advantage, goal?
* Summarize the content of the topic by mentioning subtopics, if any: in this case, X12 and JSON.

Reply:
Related to the above, you really should not need a subtitle if you have only one subtopic. In these cases, you should omit the subtitle and adjust the main title or the introduction to indicate the single topic focus.

Reply:
Got it. Thanks.